File: app/src/main/java/br/com/leonamalmeida/mobilechallenge/data/Currency.kt
package br.com.leonamalmeida.mobilechallenge.data

import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * Created by Leo Almeida on 27/06/20.
 */

@Entity
data class Currency(
    @PrimaryKey val code: String,
    val name: String
) {
    override fun toString(): String {
        return "$code - $name"
    }
}

File: mobileChallenge/Model/Currency.swift
//
//  Currency.swift
//  mobileChallenge
//
//  Created by Henrique on 03/02/25.
//

import Foundation

struct CurrencyName: Codable {
    let code: String
    let name: String
}

struct Currency: Codable {
    let currencies: [String: String]
}
